### PR TITLE
[0.3.x] CAL-405 Add ISR attributes to injections and update docs

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Add ISR attributes to injections and update docs
#### Who is reviewing it? 
(please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)
@emmberk @peterhuffer @AzGoalie 
#### Choose 2 committers to review/merge the PR.
(please choose ONLY two committers from below, delete the rest)

@bdeining
@figliold
@pklinef
@ricklarsen - Documentation

#### How should this be tested?
Same process as in https://github.com/codice/ddf/pull/2875
#### What are the relevant tickets?

[CAL-405](https://codice.atlassian.net/browse/CAL-405)

#### Checklist:
- [X] Documentation Updated
[documentation.pdf](https://github.com/codice/alliance/files/1660919/documentation.pdf)

### CODE REVIEW

File: distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 == {title}
 
-.Extended Attributes: Attributes in this group relate to intelligence, search, and reconnaissance.
+.Extended Attributes: Attributes in this group relate to intelligence, search, and reconnaissance.  *Injected by default.*
```
Comment (emmberk) [inline distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc]: ✏️ Extra space

Comment (peterhuffer) [inline distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc]: ❓ Would it be useful mention they are injected into all `MetacardType` or link to the `29.6. Developing Attribute Injections` section?

Comment (clockard) [reply distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc]: Fixed

Comment (clockard) [reply distribution/docs/src/main/resources/content/_metadataAttributes/extended-attributes-table.adoc]: There are links to that section in the CDM documentation but not sure we want links here in the tables section

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by clockard** [comment]
build now

**Comment by peterhuffer** [comment]
Successful hero. Setup CDM with override of `location.country-code=USA`, ingested a new text file, and verified the Metacard had the attribute. 👍

**Pull request merged**